src/providers/DevicesProvider/docs/AudioInputProvider.stories.mdx

```diff
@@ -4,6 +4,11 @@
 
 The `AudioInputProvider` provides a list of the user's available `audio-input` devices, the currently selected device, and the error if device selection fails.
 
+NOTE: If user chooses the default device as the current audio input device in Chrome, and plugs in a new device during the meeting that becomes the new default device, 
```

Comment:
We do not mention devices until we provide the state information in the doc, so should we move this after we explain or show the devices?

Reply:
Moved it below the `state` section